lib/active_fedora/file/attributes.rb

```diff
@@ -49,6 +49,11 @@ def create_date
     created && created.first
   end
 
+  def modified_date
+    modified = metadata.attributes["http://fedora.info/definitions/v4/repository#lastModified"]
+    modified && modified.first
```

Comment:
If there are multiple values here, mightn't we expect the most recent rather than just the first?